pkg/controller/master/node/promote_controller.go

```diff
@@ -388,15 +393,20 @@ func isPromoteStatusIn(node *corev1.Node, statuses ...string) bool {
 }
 
 func (h *PromoteHandler) createPromoteJob(node *corev1.Node) (*batchv1.Job, error) {
-	job := buildPromoteJob(h.namespace, node)
+	image, err := utilCatalog.FetchAppChartImage(h.appCache, h.namespace, releaseAppHarvesterName, []string{"generalJob", "image"})
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
NIT: can we add a log info to print image name here. It may help debug in the future. Thanks.

Reply:
Surely, I will add some logs in catalog util as well.